admin_manual/installation/apps_management_installation.rst

```diff
@@ -14,7 +14,7 @@ needs.
 Apps management
 ---------------
 
-.. figure:: /images/apps_overview.png
```

Comment:
/home/nickv/Nextcloud/Documentation/admin_manual/installation/apps_management_installation.rst:: WARNING: image file not readable: images/apps_overview.png
/home/nickv/Nextcloud/Documentation/admin_manual/installation/apps_management_installation.rst:: WARNING: image file not readable: images/apps_store.png


https://github.com/nextcloud/documentation/commit/9a54e35f8f58a69911a65abf18b1b8f64d86756f